client/components/mma/accountoverview/ManageProduct.tsx

```diff
@@ -96,6 +97,10 @@ const InnerContent = ({
 	const maybePatronSuffix =
 		productDetail.subscription.readerType === 'Patron' ? ' - Patron' : '';
 
+	const nonServiceableCountry = nonServiceableCountries.includes(
```

Comment:
Would it be better as `isNonServiceableCountry` ?

Reply:
Good suggestion!